.github/workflows/pull-request.yml

```diff
@@ -66,7 +64,6 @@ jobs:
 
   checks:
     runs-on: ubuntu-22.04
-    needs: [version]
```

Comment:
this was causing the check job to be skipped on PR to dev since version only runs on PR to master.